appinventor/components/src/com/google/appinventor/components/runtime/ImageSprite.java

```diff
@@ -138,6 +138,9 @@ public void onDraw(android.graphics.Canvas canvas) {
             // to get the other right and bottom edges
             xinit + w / 2 + rotatedBitmap.getWidth() / 2,
             yinit + h / 2 + rotatedBitmap.getHeight() / 2);
+        // The value of alpha is hardcoded to 254 to avoid the black Background that occurs
+        // after a rotation ref: issue #400
+        rotatedDrawable.setAlpha(254);
```

Comment:
A small point, pleases define that 254 as a constant and add a comment on why you used the value.  
Actually I thought Alpha will change to a float in api 11, so you might want to anticipate that (or maybe I am wrong about this).